Grab_that_article_on_Medium.py

```diff
@@ -104,19 +104,20 @@
 links = [i.get_attribute('href') for i in driver.find_elements_by_xpath("//a[@data-action-source]")]
 unique_list = list(OrderedDict(zip(links, repeat(None)))) # To remove duplicates from list
 
-f= open("output.txt","a+") # Stores ouput in output.txt in the same file directory
+f= open("output.txt","a+",encoding='utf8') # Stores ouput in output.txt in the same file directory
```

Comment:
Was the encoding necessary?

Reply:
The script is not running without the encoding on my machine. I don't know whats the default encoding. But I don't think adding the 'utf8' encoding will create any problems.

Reply:
Are you using something other than Linux or windows?
I have not checked it's working yet, so I want to know if it fails in the future.

Reply:
The script is running without encoding and with encoding also  on Mac.
```suggestion
f= open("output.txt","a+",encoding='utf8') # Stores ouput in output.txt in the same file ### directory
```